index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <link rel="stylesheet" type="text/css" href="styles.css" media="screen"/>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/9650a62e47.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="css/style.css">
+  <title>Product Landing Page - iMouse</title>
+</head>
+
+<body>
+  <header id="header">
+    <nav id="nav-bar">
+      <img id="header-img" src="https://images-na.ssl-images-amazon.com/images/I/51uTvI7ew-L._AC_SY450_.jpg" alt="logo" class="logo">
+      <ul>
+        <a class="nav-link" href="#features">Features</a>
+        <a class="nav-link" href="#how-it-work">How It Works</a>
+        <a class="nav-link" href="#pricing">Pricing</a>
+      </ul>
+    </nav>
+  </header>
+
+  <div id="container">
+    <section class="message-box">
+      <h1>Bluetooth Mouse for Apple products</h1>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+      <input id="email" type="email" name="email" placeholder="example@mail.com"><br>
+      <input id="submit" type="submit" value="Get Started" class="btn"></input>
+      </form>
+    </section>
+

```

Comment:
Please delete the extra blank line here

Reply:
- [ ] Here you can explain why is this important